### PR TITLE
Databricks Connection - User & Password clean

### PR DESCRIPTION
**Pull request by MilanBariya**
### Describe your changes :

I worked on the Databricks Connection - User & Password clean
Fix: #6828

#
### Type of change :

- [x] Bug fix
- [x] Improvement

### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] All new and existing tests passed.

#

### CODE REVIEW

**Review by ulixius9 — APPROVED** [review]

File: ingestion/setup.py [hunk 1]
```diff
@@ -176,6 +176,7 @@ def get_long_description():
         "Source": "https://github.com/open-metadata/OpenMetadata",
     },
     packages=find_namespace_packages(where="./src", exclude=["tests*"]),
+    namespace_package=["metadata"],
```
Comment (pmbrull) [inline ingestion/setup.py:179]: Could you please expand on why are we adding this?

**Review by pmbrull — COMMENTED** [review]
Thanks for checking @MilanBariya 

I believe we might be missing adding the removal on the bootstrap scripts. We need to handle those fixes on the database otherwise old connections will fail to be parsed 

Thanks!

**Review by nahuelverdugo — APPROVED** [review]
LGTM. Only one question, is this change backwards compatible?

**Comment by pmbrull** [comment]
> LGTM. Only one question, is this change backwards compatible?

It should be as those fields were not used. We just need to update the bootstrap scripts in this PR

File: ingestion/setup.py (hunk 1 above)
Comment (MilanBariya) [reply ingestion/setup.py:179]: Because of This https://github.com/open-metadata/OpenMetadata/issues/6560 Issue.

**Review by pmbrull — APPROVED** [review]
thanks for the changes! LGTM

**Pull request merged**